recipes/opensim/bld.bat

```diff
@@ -0,0 +1,21 @@
+mkdir opensim_dependencies_build
+cd .\opensim_dependencies_build
+cmake ..\dependencies^
+	-G"Visual Studio 14 2015 Win64"^
+	-DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%"
+cmake --build . --config Release -- /maxcpucount:8
+cd .. 
+
+mkdir opensim_build
+cd .\opensim_build
+cmake ..\^
+	-G"Visual Studio 14 2015 Win64"^
```

Comment:
Does conda have toggles for 32bit vs 64bit? Is there a way of telling conda the package will only work as 64bit?

Reply:
I believe we can't build 32 here but this is my only reference https://github.com/conda-forge/eofs-feedstock/issues/1

Reply:
Same here